Debug Mode and DLNA section added (#170)
jellyfin/jellyfin-docs: general/administration/connectivity.md

The easiest way to check for issues is by checking the logs, which can be accessed through the settings on the web frontend or in the log directory on your server. If there are no logs at all relating to web traffic, even over a LAN connection, then the server hasn't been reached at all yet. This would indicate either an incorrect address or an issue somewhere else on the network.

> ## Debug Mode

Reviewer: ```suggestion
## Debug (more verbose) Logging
```
?

Author: if someone doesn't know what debug means, they might not know what verbose means lol

Author: I'm still getting used to doing PR's on git. Let me know if I need to do anything to get this or my others merged.

Reviewer: This is not a debug mode per se, this is extensive logging. Debugging mode usually means your application is better suited for interoperating with a debugger tool like gdb, which is not the case here.

Author: makes sense

Reviewer: > I'm still getting used to doing PR's on git. Let me know if I need to do anything to get this or my others merged.

You have to make changes and push them to your branch for them to show up here, then we can review and merge them.

> To enable debugging for Jellyfin, it is currently required to be manually enabled. Go to Jellyfin-directory/config, in the `logging.json` file, change:

Reviewer: ```suggestion
To enable debug (much more verbose) logging for Jellyfin, it is currently required to manually edit config files - no UI options exist yet. Go to Jellyfin-directory/config, in the `logging.json` file, change:
```

`"MinimumLevel": "Information",`

to

`"MinimumLevel": "Debug",`

Jellyfin 10.4.1 and above will auto-reload the new configuration. The debug messages show up in the log with the tag DBG.

## DLNA

> DLNA is based on uPnP. DLNA will send a broadcast signal from Jellyfin. This broadcast is limited to Jellyfin's current subnet. If using Docker Host Mode is required, otherwise the broadcast signal will only be sent in the bridged network inside of docker. 

Reviewer: ```suggestion
DLNA is based on uPnP. DLNA will send a broadcast signal from Jellyfin. This broadcast is limited to Jellyfin's current subnet. If using Docker, network in Host Mode is required, otherwise the broadcast signal will only be sent in the bridged network inside of docker. 
```
